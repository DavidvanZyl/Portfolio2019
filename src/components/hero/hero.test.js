import React from "react"
import Hero from "./hero"

describe("BlogEntry", () => {
  test("renders correctly", () => {
    const wrapper = shallow(
      <Hero item={{ title: "test", image: "test.jpg", alt: "test image" }} />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
