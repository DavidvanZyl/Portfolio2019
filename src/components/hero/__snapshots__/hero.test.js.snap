// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogEntry renders correctly 1`] = `
<div
  className="hero"
>
  <div
    className="filtered"
  >
    <img />
  </div>
  <div
    className="hero__titleWrap"
  >
    <h1
      className="hero__title"
    />
  </div>
</div>
`;
